### PR TITLE
Uart handlers added

### PR DESCRIPTION
**Pull request by jrachucki-n7s**

### CODE REVIEW

File: src/hwas/hwas.c [hunk 1]
```diff
@@ -178,6 +178,41 @@ MCAN1_Handler(void)
     }
 }
 
+void
```
Comment (Lurkerpas) [inline src/hwas/hwas.c:182]: How does it affect using TASTE Uart driver?

Comment (jrachucki-n7s) [reply src/hwas/hwas.c:182]: Fixed in new commit

File: src/Hal/Hal.c [hunk 2]
```diff
@@ -15,6 +14,14 @@
 
 static sXdmad xdmad;
 
+static volatile InterruptCallback* interruptSubscribe[Nvic_InterruptCount] = { NULL };
```
Comment (Lurkerpas) [inline src/Hal/Hal.c]: Maybe interruptSubsciption?

Comment (jrachucki-n7s) [reply src/Hal/Hal.c]: sure!

Comment (jrachucki-n7s) [reply src/Hal/Hal.c]: Fixed in new commit

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**